lib/views/mentor/mentorscreen.dart

```diff
@@ -36,7 +30,7 @@ class _MentorScreenState extends State<MentorScreen> {
         children: [
           MentorHome(),
           Text("Second Screen for Mentor", textScaleFactor: 2),
-          Text("Third Screen for Mentor", textScaleFactor: 2),
+          MentorChatScreen(),
```

Comment:
Could you move this here: https://github.com/PrepHQ/PrepHQ_Connect/blob/master/lib/views/student/mentorprofilescreen.dart#L71

Reply:
I dont understand what you mean

Reply:
can you move the chat from being the resource screen to being the screen that pops up when you hit the chat button in the mentorprofilescreen

Reply:
So this one is the Mentor's view of the chat, not the Student's view of the chat, right? So it should stay as he has it.

Reply:
My mistake - I meant the student's view of the chat. that one could be changed back now.

Regardless, the test would need to be fixed to look for the chat window in the resources tab in the mentor view